src/components/Modals/NewCourse/NewCourseModal.vue

```diff
@@ -83,7 +83,6 @@ export default Vue.extend({
       selfCheckRequirements: [] as readonly RequirementWithIDSourceType[],
       editMode: false,
       courseSelectorKey: 0,
-      courseIsAddable: true,
```

Comment:
The only places where `courseIsAddable` is ever set to false are the deleted uncalled code. Now that they are gone, we know it's always true.